### PR TITLE
Examine management patch

### PR DESCRIPTION
**Pull request by jonat123**
Updated the article for V10

### CODE REVIEW

File: Reference/Searching/Examine/examine-management.md [hunk 1]
```diff
@@ -1,49 +1,38 @@
...
 
-:::note
-This document has been verified for Umbraco 7.
-
-If you are using Umbraco 8 or later versions, please refer to the note on the [Examine documentation landing page](index.md) for more details.
-:::
-
 ## Overview
 
 The Umbraco backoffice allows you to view details about your examine indexes and searches along with those that power the internal CMS search all in one place. You can see which fields are being indexed, rebuild the indexes if there's a problem and even test keywords to see what results would be returned.
```
Comment (Jay-umbr) [inline Reference/Searching/Examine/examine-management.md:11]:
```suggestion
The Umbraco backoffice allows you to view details about your Examine indexes and searchers - all in one place. You can see which fields are being indexed, rebuild the indexes if there's a problem, and test keywords to see what results would be returned.
```

File: Reference/Searching/Examine/examine-management.md [hunk 2]
```diff
@@ -1,49 +1,38 @@
...
 
-![Examine Management within the Developer section](images/examine-management-home.png)
+![Examine Management within the Developer section](images/overview-examine.png)
 
-The Examine Management section, accessible from within the Developer area, is split into two sections: Indexers and Searchers.
+The Examine Management section, accessible from within the Settings section, is split into two sections: Indexers and Searchers.
 
 ## Indexers
 
 From the Indexers section, you can view details about each Examine index currently configured within your Umbraco installation. Selecting any of these will expand to show you four additional options, each discussed below.
```
Comment (Jay-umbr) [inline Reference/Searching/Examine/examine-management.md:19]:
```suggestion
From the Indexers section, you can view details about each Examine index currently configured within your Umbraco installation. Clicking any of these indexes will show you additional options, each discussed below.
```

File: Reference/Searching/Examine/examine-management.md [hunk 3]
```diff
@@ -1,49 +1,38 @@
...
 
-### Index info & tools
-
-This section allows you to see the high level statistics about the index that you selected, including how many documents and fields are currently being stored.
-
-![Rebuild Index within Examine Management](images/examine-management-rebuild-index.png)
+### Index info
 
-This section also provides the ability to rebuild the index should this be required. Depending on how much content your website has, rebuilding the search indexes could take a while and affect the site performance temporarily, so it is not recommended to do this while the website is under high load.
+This section allows you to see the list of properties about the index that you selected, including how many documents and fields are currently being stored.
```
Comment (Jay-umbr) [inline Reference/Searching/Examine/examine-management.md:23]:
```suggestion
This section allows you to see the list of properties on the index that you selected, including how many documents and fields are currently being stored.
```

File: Reference/Searching/Examine/examine-management.md [hunk 4]
```diff
@@ -1,49 +1,38 @@
...
+![Rebuild Index within Examine Management](images/External-indexes-v10.png)
 
-From here you can see the default system fields that are stored for each document within the search index, including whether sorting is enabled for each field and what datatype the data is stored as.
+Within the Indexers it displays the details for the index provider as well.
 
-### User fields
+This can be useful to confirm the configuration that Umbraco is using and to ensure it is working as expected. This section also displays the full file path of the index itself.
 
-This section is exactly the same as system fields, but with any custom fields being listed instead.
+This section also provides the ability to rebuild the index should this be required. Depending on how much content your website has, rebuilding the search indexes could take a while and affect the site performance temporarily, so it is not recommended to do this while the website is under high load.
```
Comment (Jay-umbr) [inline Reference/Searching/Examine/examine-management.md:31]:
```suggestion
This section also provides the ability to rebuild the index, should this be required. Depending on how much content your website has, rebuilding the search indexes could take a while and affect the site performance temporarily, so it is not recommended to do this while the website is under high load.
```

File: Reference/Searching/Examine/examine-management.md [hunk 5]
```diff
@@ -1,49 +1,38 @@
...
 
-### Provider properties
+### fields
```
Comment (Jay-umbr) [inline Reference/Searching/Examine/examine-management.md:33]:
```suggestion
### Fields
```

File: Reference/Searching/Examine/examine-management.md [hunk 6]
```diff
@@ -1,49 +1,38 @@
...
 
-The final section within Indexers displays the details for the index provider. This can be useful to confirm the configuration that Umbraco is using and to ensure it is working as expected. This section also displays the full file path of the index itself.
+From here you can see the default system fields that are stored for each document within the search index, including the number of fields document, The score which is calculated by Examine depending on how closely the individual results matched the search term.
```
Comment (Jay-umbr) [inline Reference/Searching/Examine/examine-management.md:35]:
```suggestion
From here you can see the default system fields that are stored for each document within the search index. That includes the number of fields document, and the score which is calculated by Examine depending on how closely the individual results matched the search term.
```

**Review by Jay-umbr — CHANGES_REQUESTED** [review]
Looks like everything is correct though I would ask you to take a look at my suggestions. Mostly phrasing, grammas and some missing commas

**Comment by jonat123** [comment]
@Jay-umbr thanks! 

I have added your suggestions and will merge it in :D

**Pull request merged**